src/v/cluster/types.h

```diff
@@ -981,18 +981,21 @@ struct remote_topic_properties
     remote_topic_properties() = default;
     remote_topic_properties(
       model::initial_revision_id remote_revision,
-      int32_t remote_partition_count)
+      int32_t remote_partition_count,
+      std::optional<s3::bucket_name> bucket)
       : remote_revision(remote_revision)
-      , remote_partition_count(remote_partition_count) {}
+      , remote_partition_count(remote_partition_count)
+      , bucket_name(std::move(bucket)) {}
 
     model::initial_revision_id remote_revision;
     int32_t remote_partition_count;
+    std::optional<s3::bucket_name> bucket_name;
 
     friend std::ostream&
     operator<<(std::ostream&, const remote_topic_properties&);
 
     auto serde_fields() {
-        return std::tie(remote_revision, remote_partition_count);
+        return std::tie(remote_revision, remote_partition_count, bucket_name);
```

Comment:
does the version need to be bumped or is it all still at v0?

Reply:
Read replicas are not released yet, but I updated the version just in case.

Reply:
Let's not update the version unless we need to. It sounds like it is ok to leave it at v0.

Reply:
fixed that